plugin/storage/kafka/options.go

```diff
@@ -94,3 +181,36 @@ func (opt *Options) InitFromViper(v *viper.Viper) {
 func stripWhiteSpace(str string) string {
 	return strings.Replace(str, " ", "", -1)
 }
+
+// getCompressionLevel to get compression level from compression type
+func getCompressionLevel(compressionMode string, compressionLevel int) (int, error) {
+	compressionModeData := compressionModes[compressionMode]
```

Comment:
in the other functions you use ToLower, should it be consistent?

Reply:
fixed